_posts/2020-07-08-pull-requests-require-context.md

```diff
@@ -0,0 +1,148 @@
+you know you've forgotten to include context. It doesn't take much time for you to add the missing context to keep
+the conversation moving forward.
+
+It takes a lot longer to identify missing context when communicating asynchronously. Worse, lack of context when
+_reviewing code_ asynchronously has a reverb effect. I create my PR when it's convenient for me, you ask a
+clarifying question when it's convenient for you, I respond when it's convenient for me, etc. Suddenly my PR has
+been open for three days and we haven't yet made it to a common understanding of why I've made these changes.
+
+It's extremely important to include all available context when drafting a PR. It saves incredible amounts of time
+by cutting out slow round-trip conversations to clarify.
+
+I'm personally proud of and impressed by the job we do at Artsy in including context in our PRs. We start early, by
+giving our engineers
+[some reading about how we work with PRs during their onboarding](https://github.com/artsy/README/blob/master/playbooks/engineer-workflow.md#pull-requests).
+
+But beyond that our engineers lead by example. This article presents a handful of examples from Artsy repositories
+demonstrating how you can add context to your PRs to avoid unnecessary clarifying conversation.
+
+## Explain Your Reasoning
+
+You've been thinking a lot about the problem you're solving - probably significantly more than your reviewers.
+You'll save everyone time by describing the problem and sharing how you're thinking about it.
+
+### Define the problem and solution
+
+Why does this PR exist? Explain the problem it solves and describe your solution, as
+[Sarah](https://github.com/sweir27) does [in this PR](https://github.com/artsy/force/pull/3095). For bonus points,
+include alternative approaches you considered.
+
+### Explain interesting lines of code
+
+The reviewers aren't the only ones who can comment on lines of code.
+[Give them](https://github.com/artsy/emission/pull/2085#discussion_r378228269)
+[additional information](https://github.com/artsy/emission/pull/2085#discussion_r378230196)
+[about why](https://github.com/artsy/emission/pull/2085#discussion_r378231974) a particular line was written, as
+[David](https://github.com/ds300) does [in this PR](https://github.com/artsy/emission/pull/2085). Maybe you want
+feedback focused on that line or maybe the line has side-effects and implications that aren't obvious.
+
+### Give a guided tour of the changes
+
+[Devon](https://github.com/dblandin) takes the idea of adding context to individual lines to the next level
+[in this PR](https://github.com/artsy/reaction/pull/2774#pullrequestreview-288095754). He takes advantage of
+markdown to give us a virtual tour of the changes, at each stop providing helpful information and a link to the
+next change. It's like he's sitting next to you!
+
+## Show Your Work
+
+If your PR contains work that is beyond trivial, "showing your work" can help reviewers understand what they're
```

Comment:
What do you mean by "show your work"? This could use one more sentence to provide an example and explain what exactly you mean!